Tests/ParseSwiftTests/KeychainStoreTests.swift

```diff
@@ -70,56 +69,56 @@ class KeychainStoreTests: XCTestCase {
         XCTAssertNil(testStore[bool: key], "Values should be equal after get")
     }
 
-    func testSetComplextObject() {
-        let complexObject: [Any] = [["key": "value"], "string2", 1234, NSNull()]
-        testStore["complexObject"] = complexObject
-        guard let retrievedObject: [Any] = testStore["complexObject"] else {
-            return XCTFail("Should retrieve the object")
-        }
-
-        XCTAssertTrue(retrievedObject.count == 4)
-        retrievedObject.enumerated().forEach { (offset, retrievedValue) in
-            let value = complexObject[offset]
-            switch offset {
-            case 0:
-                guard let dict = value as? [String: String],
-                    let retrivedDict = retrievedValue as? [String: String] else {
-                        return XCTFail("Should be both dictionaries")
-                }
-                XCTAssertTrue(dict["key"] == retrivedDict["key"])
-            case 1:
-                guard let string = value as? String,
-                    let retrievedString = retrievedValue as? String else {
-                        return XCTFail("Should be both strings")
-                }
-                XCTAssertTrue(string == retrievedString)
-            case 2:
-                guard let int = value as? Int,
-                    let retrievedInt = retrievedValue as? Int else {
-                        return XCTFail("Should be both ints")
-                }
-                XCTAssertTrue(int == retrievedInt)
-            case 3:
-                guard let retrieved = retrievedValue as? NSNull else {
-                        return XCTFail("Should be both ints")
-                }
-                XCTAssertTrue(retrieved == NSNull())
-            default: break
-            }
-        }
-    }
+    //MARK:- Todo
+//    func testSetComplextObject() {
+//        let complexObject: [Any] = [["key": "value"], "string2", 1234, NSNull()]
```

Comment:
How about use AnyCodable for complexObject?

Reference : https://github.com/Flight-School/AnyCodable

Reply:
Why not? Seems like a decent idea.

Reply:
Thanks, I'm going to use AnyCodable in Parse-Swift Project.